app/controllers/users.js

```diff
@@ -38,16 +38,20 @@ const passwordValidation = (password) => {
   }
 };
 
-exports.signup = (req, res, next) => {
-  const newUser = req.body;
-  emailValidation(newUser.email);
-  passwordValidation(newUser.password);
-  if (!newUser.firstName) {
+const userValidation = (user) => {
+  if (!user.firstName) {
     throw errors.validationError('The firstName is missing');
   }
-  if (!newUser.lastName) {
+  if (!user.lastName) {
     throw errors.validationError('The lastName is missing');
   }
+};
+
+exports.signup = (req, res, next) => {
+  const newUser = req.body;
+  emailValidation(newUser.email);
+  passwordValidation(newUser.password);
```

Comment:
you can put this too function calls inside the `userValidation` and avoid repeting them too.